src/api/immutable.js

```diff
@@ -94,8 +94,9 @@ class Writer extends helpers.NetworkObject {
   /**
   * Proxy to `close`.
   */
-  save() {
-    return this.close();
+  async save() {
```

Comment:
I would try to keep a consistency in the code to either use `Promises` or `async`, so we could plan for a refactoring to convert all the code to make use of `async` but until then I would say we stick to `Promises` to have a consistency in the code style.

Reply:
Moreover, we can get rid of this function as it's not providing anything but just a name proxy, also I'm not sure if we want to have a function which by default stores data unencrypted, this could be error prone for the end users.